dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -4066,6 +4074,27 @@ paths:
               $ref: '#/components/schemas/Permission'
         description: user permission
         required: true
+  '/workflows/path/workflow/{repository}/published':
```

Comment:
FYI, this isn't actually a new endpoint. Rather, github diff isn't smart enough to know that the above endpoint was inserted rather than changed.